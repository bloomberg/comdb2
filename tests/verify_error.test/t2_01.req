1 create table test1 (i int primary key, j int)
2 create table test2 (i int primary key, j int)
1 insert into test1 values (1, 1)
2 insert into test2 values (1, 1)

1 begin
1 update test1 set j = j + 1
2 update test1 set j = j + 1
1 commit
1 select * from test1

1 begin
1 update test1 set j = j + 1
2 update test1 set j = j + 1
1 truncate test2
1 commit
1 select * from test1
1 select * from test2

2 update test1 set j = j - 1
1 begin
1 truncate test2
1 update test1 set j = j + 1
2 update test1 set j = j + 1
1 commit
1 select * from test1
1 select * from test2

1 begin
1 update test1 set j = j + 1
2 update test1 set j = j + 1
1 commit
1 select * from test1

1 begin
1 update test1 set j = j + 1
1 truncate test2
2 create index idx on test2(j)
1 commit
1 select * from test1
1 select * from test2
