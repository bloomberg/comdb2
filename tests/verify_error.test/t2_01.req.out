1 [create table test1 (i int primary key, j int)]
done
2 [create table test2 (i int primary key, j int)]
done
1 [insert into test1 values (1, 1)]
(rows inserted=1)
done
2 [insert into test2 values (1, 1)]
(rows inserted=1)
done
1 [begin]
done
1 [update test1 set j = j + 1]
done
2 [update test1 set j = j + 1]
(rows updated=1)
done
1 [commit]
done
1 [select * from test1]
(i=1, j=3)
done
1 [begin]
done
1 [update test1 set j = j + 1]
done
2 [update test1 set j = j + 1]
(rows updated=1)
done
1 [truncate test2]
done
1 [commit]
[commit] failed with rc 2 find old record failed unable to update record rc = 4
done
1 [select * from test1]
(i=1, j=4)
done
1 [select * from test2]
(i=1, j=1)
done
2 [update test1 set j = j - 1]
(rows updated=1)
done
1 [begin]
done
1 [truncate test2]
done
1 [update test1 set j = j + 1]
done
2 [update test1 set j = j + 1]
(rows updated=1)
done
1 [commit]
[commit] failed with rc 2 find old record failed unable to update record rc = 4
done
1 [select * from test1]
(i=1, j=4)
done
1 [select * from test2]
(i=1, j=1)
done
1 [begin]
done
1 [update test1 set j = j + 1]
done
2 [update test1 set j = j + 1]
(rows updated=1)
done
1 [commit]
done
1 [select * from test1]
(i=1, j=6)
done
1 [begin]
done
1 [update test1 set j = j + 1]
done
1 [truncate test2]
done
2 [create index idx on test2(j)]
done
1 [commit]
[commit] failed with rc 240 stale version for table:test2 master:1 replicant:0
done
1 [select * from test1]
(i=1, j=6)
done
1 [select * from test2]
(i=1, j=1)
done
